src/config/chain.d.ts

```diff
@@ -26,6 +26,19 @@ export const CHAIN_ID: Record<ChainName, ChainId> = {
   VOLTA: '73799',
 }
 
+export const CHAIN_PREFIXES: Record<string, ChainId> = {
```

Comment:
How do I access the loaded ones at the time getInitialChain is called?

Reply:
Can you not set the initial chain after the chains have loaded?

Reply:
I've realized we already had this: https://github.com/gnosis/safe-react/blob/dev/src/index.tsx#L11-L13
But it stopped working because `getChains()` doesn't have anything at the moment it's called.